src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -97,6 +97,9 @@ public function load(array $configs, ContainerBuilder $container)
         // paths are modified in ExtensionPass if forms are enabled
         $container->getDefinition('twig.template_iterator')->replaceArgument(1, $config['paths']);
 
+        $container->getDefinition('twig.template_iterator')->replaceArgument(3, $config['file_name_pattern']);
+        $container->getDefinition('twig.command.lint')->replaceArgument(1, $config['file_name_pattern'] ?: ['*.twig']);
```

Comment:
Should the default be "*" for BC? If we keep "*.twig" as a default, this should at least be documented in the CHANGELOG file.

Reply:
For the linter, I kept `*.twig` for BC.